application/build.gradle

```diff
@@ -0,0 +1,119 @@
+plugins {
+    // Spring
+    id 'org.springframework.boot' version '2.6.6'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+
+    // Sonarcloud
+    id "org.sonarqube" version "3.3"
+    id "argelbargel.gradle.plugins.sonarqube-multiproject-plugin" version "1.1"
+
+    // Jacoco for sending code coverage to Sonarcloud
+    id 'jacoco-report-aggregation'
+
+    id 'java'
+
+    //pmd
+    id 'pmd'
+
+    //checkstyle
+    id 'checkstyle'
+}
+
+group 'com'
+version '0.0.1-SNAPSHOT'
+sourceCompatibility = '18'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':core')
+    implementation project(':adapters:outgoing:jpa')
+    implementation project(':adapters:incoming:rest')
+
+    // Spring
+    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
```

Comment:
application shouldnt need starter web

and shouldnt need most of the dependencies below here
(javax, oauth2 client, AD, security)